server_admin/topics/login-settings/forgot-password.adoc

```diff
@@ -1,18 +1,33 @@
 
 == Enabling forgot password
 
-If you enable `Forgot Password`, users can reset their login credentials if they forget their passwords or lose their OTP generator. 
+If you enable `Forgot password`, users can reset their login credentials if they forget their passwords or lose their OTP generator. 
 
 .Procedure
+ifeval::[{project_product}==true]
 . Click *Realm Settings* in the menu.
+endif::[]
+ifeval::[{project_community}==true]
+. Click *Realm settings* in the menu.
+endif::[]
 . Click the *Login* tab.
 +
 .Login tab
 image:{project_images}/login-tab.png[Login Tab]
```

Comment:
This screen shot is for the new console, but it also appears in the old console documentation.